flatpak-extension-runtime-fdo-openh264-watcher.path

```diff
@@ -0,0 +1,8 @@
+[Unit]
+After=systemd-tmpfiles-setup.service
+
+[Path]
+PathChanged=/var/lib/flatpak/runtime/org.freedesktop.Platform.openh264/x86_64/2.0/active/files/extra/libopenh264.so.6
```

Comment:
Just a note that I tried watching files under /run/lib/openh264 instead but in this case it would only trigger the unit when installing the runtime but not when uninstalling it.

Also tried watching the dir itself, but that is racy as the unit may be triggered before all files are written to the dir.

Reply:
In the tmpfiles snippet you're substituting `@FLATPAK_ARCH@`, but here `x86_64` is hardcoded.

Reply:
Good catch, should be fixed now.